.github/workflows/test-runner-pr-integration.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         ./render-k8s-config.sh wsmtest ${{ steps.vault-token-step.outputs.vault-token }}
         echo "Running test suite"
         export TEST_RUNNER_SERVER_SPECIFICATION_FILE="workspace-local.json"
-        ./gradlew runTest --args="suites/BasicIntegration.json /tmp/TestRunnerResults"
+        ./gradlew runTest --args="suites/FullIntegration.json /tmp/TestRunnerResults"
```

Comment:
How much longer is this taking than the basic suite? I thought the plan was for nightly to be the long-running one and this one to be quicker, but it looks like they're doing the same suite now.

Reply:
The basic suite is ~5m total, and the new suite is 8-10m. To me that's getting close to the limit of a presubmit test, but I think it's worth the additional coverage (especially since our connected tests are already pushing 15 minutes.

Reply:
Yeah, we might want to consolidate at some point. I think the sweet spot is under 3 minutes if we can get back down.